### DISCUSSION TITLE
gmc/blog/community-news/introducing-community-posts/

### DISCUSSION THREAD

**Discussion by giscus** [Gothic-Modding-Community/gmc#130] [_]
# gmc/blog/community-news/introducing-community-posts/

Gothic Modding Community is a collection of helpful resources for modding Gothic and Risen games.

http://127.0.0.1:8000/gmc/blog/community-news/introducing-community-posts/

**Comment by kamilkrzyskow** [comment]
Great feature, can't wait to use it when it's released :v:
